mmaction/models/backbones/resnet3d.py

```diff
@@ -50,16 +47,15 @@ def __init__(self,
                  downsample=None,
                  style='pytorch',
                  inflate=True,
-                 inflate_style='3x1x1',
                  non_local=False,
                  non_local_cfg=dict(),
                  conv_cfg=dict(type='Conv3d'),
                  norm_cfg=dict(type='BN3d'),
                  act_cfg=dict(type='ReLU'),
-                 with_cp=False):
+                 with_cp=False,
+                 **kwargs):
```

Comment:
where is this `kwargs` used?

Reply:
To absorb unused `inflate_style` argument

Reply:
then explicitly check it?

Reply:
nonono, only `inflate_style` is allowed in `kwargs`

Reply:
How about just keeping the `inflate_style` and mentioning that it is unused in `BasicBlock3d`. Otherwise it is wired to check the kwargs, since it contributes nothing for the class

Reply:
`set(kwargs.keys()).issubset(set(['inflate_style ']))`

we want to make sure that only inflate_style is passed into kwargs, so that typos like `activation_cfg` could be captured

Reply:
forgot assert